back-end/stagram/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+# Create your models here.
+
+
+class User(models.Model):
+    username = models.CharField(max_length=128)
+    password = models.CharField(max_length=25)
+    email = models.CharField(max_length=50)
+
+    def __str__(self):
+        return f'{self.username}-{self.email}'
+
+
+class Photo(models.Model):
+    name = models.CharField(max_length=128)
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name='user')
+
+    def __str__(self):
+        return self.name
+
+
+class Profile(models.Model):
+    bio = models.TextField()
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name='user')
```

Comment:
We'll want to create a `OneToOne` relationship from our Profile model to our User model (not a `ForeignKey`, which implies a one-to-many relationship)

Reply:
Here's an example from Django docs which extends an `Employee` model from the pre-made `User` model: https://docs.djangoproject.com/en/4.0/topics/auth/customizing/#extending-the-existing-user-model

Reply:
I changed the relationship between profile and user model from one-to-many to one-to-one.